.github/workflows/append_release.yml

```diff
@@ -17,7 +17,7 @@ jobs:
       - name: Setup node
         uses: actions/setup-node@v2
         with:
-          node-version: "12"
+          node-version: "16"
```

Comment:
We were on node 12, which was pretty old. The update process was nice and smooth once I updated webpacker.